### PR TITLE
Adds LIGO-Hanford-CE1

### PR DESCRIPTION
**Pull request by ashtongraves**

### CODE REVIEW

File: topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml [hunk 1]
```diff
@@ -149,4 +149,22 @@ Resources:
...
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Ashton Graves
+          ID: OSG1000362
+      Security Contact:
+        Primary:
+          Name: Ashton Graves
+          ID: OSG1000362
```
Comment (brianhlin) [inline topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml:162]: Can you add some backup contacts?

File: topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml [hunk 2]
```diff
@@ -149,4 +149,22 @@ Resources:
...
+    FQDN: ligo-hanford-ce1.svc.opensciencegrid.org
+    ID: 1480
```
Comment (brianhlin) [inline topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml:164]:
```suggestion
```

IDs are now automatically generated

**Review by brianhlin — APPROVED** [review]
Some requests. Pre-approving

File: topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml (hunk 1 above)
Comment (ashtongraves) [reply topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml]: db4e6b3444457a992002bfe70c495343efab1d27 resolves

**Comment by ashtongraves** [comment]
@brianhlin can I delete this pr?

**Pull request closed without merging**